src/Functional.CQS.AOP.CommonTestInfrastructure.MetricsCapturing/Functional.CQS.AOP.CommonTestInfrastructure.MetricsCapturing.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <DocumentationFile>bin\Release\Functional.CQS.AOP.CommonTestInfrastructure.MetricsCapturing.xml</DocumentationFile>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>bin\Debug\Functional.CQS.AOP.CommonTestInfrastructure.MetricsCapturing.xml</DocumentationFile>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
Looks like `TreatWarningsAsErrors` was only enabled for the `Debug` configuration in this project? I think the `Release` config was missed.

Reply:
Looking at https://github.com/RyanMarcotte/Functional.CQS/commit/6a443c85133fa894d5cba0e19f65df05b7f73d55, it looks like this comment applies to all the Tests projects.

Reply:
Haha, oops.  Will fix.

Reply:
That will be the case for all the projects I made modifications to